tests/fixtures/r_transform_fit_custom_sparse_input.R

```diff
@@ -2,14 +2,15 @@
 #
 # All rights reserved.
 #
-# 
+#
 #
 # This is proprietary source code of DataRobot, Inc. and its affiliates.
 #
 # Released under the terms of DataRobot Tool and Utility Agreement.
 fit <- function(X, y, output_dir, class_order=NULL, row_weights=NULL, ...){}
 
 transform <- function(X, transformer, y=NULL, ...){
+    stopifnot(!is.null(colnames(X)))
```

Comment:
lets add an assert like the python test where it inspects the colname values

Reply:
OK, assert has been fixed.  Can you approve?